test/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+NAME="$(uname -s)"
+RELEASE="$(uname -r)"
+
+cd "$(dirname "$0")/.."
+echo
+
+if ([[ "$RELEASE" = *'boot2docker' ]] || [[ "$RELEASE" = *'moby' ]]) && \
```

Comment:
Moby is the name of Docker's whale mascot/logo so I feel confident it's Docker only.  It seems to be in `$RELEASE` since the new Docker for Mac dropped VirtualBox.